lib/queries.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
       "access_tokens WHERE access_token = $1;"
   },
   create: {
-    client: "INSERT INTO clients VALUES($1, $2, $3, $4);",
+    client: "INSERT INTO clients VALUES($1, $2, $3, $4, $5);",
```

Comment:
Could you name these columns explicitly? I worry about the case where the db column order isn't the same as what this code is expecting. If you wanna fix up the two lines below this too that'd be lovely :)